src/components/Book.js

```diff
@@ -1,13 +1,21 @@
 import React from 'react';
 import Proptypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { del } from '../redux/books/books';
 
 export default function Book({ book }) {
+  const dispatch = useDispatch();
   return (
     <div className="bookCard">
       <div className="">
+        <p className="genre">{book.genre}</p>
```

Comment:
- [ ] The page is not rendered check the issue below  the issue  The prop `books` is marked as required in `Books`, but its value is `undefined`
![dgf](https://user-images.githubusercontent.com/44745855/163240821-ae48d57b-cc9b-4f27-be6e-512421886562.png)